### PR TITLE
Dockerize to deploy with shinyproxy

### PR DESCRIPTION
**Pull request by doug-leasure**
Added a `pkg_build.R` script that uses golem to create a Dockerfile.  

The dockerized app can be deployed using shinyproxy so that the app is more scalable because each user gets their own R environment.  This avoids the need for a Shiny Server Pro license. I successfully tested the app container with shinyproxy running behind nginx as a reverse proxy web server.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -30,15 +30,16 @@ Imports:
   tidyr,
   tibble,
   shiny,
+  shinydashboard,
```
Comment (mpascariu) [inline DESCRIPTION:33]: Probably we don't need to import {shinydashboard}, do we? 
We are dependent already on way to many libraries.

**Review by mpascariu — CHANGES_REQUESTED** [review]
Thanks @doug-leasure, docker is the way to go!
Not sure that we need to import {shinydashboard} package, do we? Apart from this the pr looks good to me.

**Comment by doug-leasure** [comment]
Hi @mpascariu, I think the `shinydashboard` dependency is coming from line 179 in `app_ui.R`.  The docker container returns an error from the app if shinydashboard is not included in the Imports.

**Comment by mpascariu** [comment]
Ok. Let's leave it like this then, for now.

**Pull request merged**